views/mailinglists.ejs

```diff
@@ -2,66 +2,17 @@
 <% title = 'OpenMRS ID - Mailing Lists' %>
 <% headline = 'Mailing Lists' %>
 <% headAppend = '<link rel="stylesheet" href="/mailinglists/resource/mailinglists.css">' %>
-<% sidebar = ['./../views/sidebar/groups-secondaryemail'] %>
```

Comment:
I think you need to include your sidebar content that you have in `views/sidebar.ejs` here. Without this sidebar variable set, the page looks like this.
![gci-39-nosidebar](https://cloud.githubusercontent.com/assets/910198/5690084/93695bcc-9843-11e4-9075-78e7bb8f52b6.jpg)